src/modules/data/useData.ts

```diff
@@ -0,0 +1,18 @@
+import { useFirestore, useFirestoreCollectionData } from 'reactfire';
+
+const useData = <T, D>(COLLECTION_NAME: string) => {
+  const dataRef = useFirestore().collection(COLLECTION_NAME);
+  const data = useFirestoreCollectionData<T>(dataRef, {
+    idField: 'id',
+  });
+  const addData = async (d: D) => {
+    const creationDate = new Date();
+    return await dataRef.add({ ...d, creationDate });
+  };
+  const removeData = async (id: string) => {
```

Comment:
What do u think of `removeDoc` and `addDoc`?

Reply:
Makes sense to me